.github/workflows/tests.yml

```diff
@@ -84,7 +84,7 @@ jobs:
 
       - name: Archive logs artifacts
         if: ${{ failure() }}
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
         with:
           name: logs_php-${{ matrix.php }}
```

Comment:
the name needs to be update to also include other elements of the matrix, to avoid having multiple jobs uploading with the same name, which is forbidden in v4

Reply:
Done.

The build fails. Not sure if it passed in recent couple of years, while GitHub Actions was preserving past build logs.